cmd/livepeer/livepeer.go

```diff
@@ -160,23 +166,60 @@ func main() {
 	if err != nil {
 		glog.Errorf("Error creating livepeer node: %v", err)
 	}
+	if *transcoder {
+		n.NodeType = core.Transcoder
+	} else if *bootnode {
+		n.NodeType = core.Bootnode
+	} else if *gateway {
+		n.NodeType = core.Gateway
+	} else {
+		n.NodeType = core.Broadcaster
+	}
 
-	if *bootnode {
+	if *bootnode || *transcoder || *gateway {
 		glog.Infof("\n\nSetting up bootnode")
```

Comment:
Update log message since we could be starting up as a bootnode, transcoder or gateway node now?